### PR TITLE
update request vars to latest aiohttp

### PR DESCRIPTION
**Pull request by doubledare704**
Changes deprecated POST and GET to new post() coroutine and query property

### CODE REVIEW

**Comment by jettify** [comment]
Thanks! could you please fix flake8 error?
```
aiohttp_debugtoolbar/panels/request_vars.py:30:80: E501 line too long (81 > 79 characters)
```

**Comment by doubledare704** [comment]
@jettify ok

**Comment by doubledare704** [comment]
# It falls
[https://github.com/doubledare704/aiohttp-debugtoolbar/blob/master/tests/test_middleware.py#L177](url)
assert awaits 200 but resp returns 500

File: requirements-dev.txt [hunk 1]
```diff
@@ -8,7 +8,7 @@ aiohttp-jinja2==0.13.0
 aiohttp-mako==0.3.0
 docutils==0.13.1
 pygments==2.2.0
-aiohttp==1.3.3
+aiohttp==2.0.2
```
Comment (jettify) [inline requirements-dev.txt]: lets remove version bump for now

**Comment by jettify** [comment]
It is expected, anyway your PR is useful, please remove aiohttp version bump so we can merge it.

**Comment by jettify** [comment]
Thanks!  (tests is fixed in master)

**Pull request merged**